nominees/quentb.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "QuenTB",
```

Comment:
The name (and associated file) should be renamed to 'QuanTB'

Reply:
Fixed in e9bb17d